lib/DDG/Spice/Crackhash.pm

```diff
@@ -0,0 +1,30 @@
+package DDG::Spice::Crackhash;
+
+# ABSTRACT: Write an abstract here
+
+# Start at http://docs.duckduckhack.com/walkthroughs/forum-lookup.html if you are new
+# to instant answer development
+
+use DDG::Spice;
+use strict; 
+
+# Caching - http://docs.duckduckhack.com/backend-reference/api-reference.html#caching
+spice is_cached => 0;
+spice proxy_cache_valid => "200 1d"; # defaults to this automatically
+
+spice wrap_jsonp_callback => 0; # only enable for non-JSONP APIs (i.e. no &callback= parameter)
+
+# API endpoint - http://docs.duckduckhack.com/walkthroughs/forum-lookup.html#api-endpoint
+spice to => 'http://api.md5crack.com/crack/ZFCQCcpS1kLgbCQZ/$1';
+
+# Triggers - http://docs.duckduckhack.com/walkthroughs/forum-lookup.html#triggers
+triggers any => 'Crackmyhash', 'Crackhash' , 'crack hash';
```

Comment:
@flipchan I don't think the `Crackmyhash` and `Crackhash` triggers are going to be that useful.

`crack hash` is fine, and then you could use `crack` as well.

Does this only work on `MD5` hashes? If so, we can perform a quick filter in the `handle` to make sure the query matches a valid hash; this would save quite a few unnecessary API calls.